fairlearn/reductions/_moments/conditional_selection_rate.py

```diff
@@ -2,11 +2,15 @@
 # Licensed under the MIT License.
 
 import pandas as pd
+import numpy as np
 from .moment import ClassificationMoment
-from .moment import _GROUP_ID, _LABEL, _PREDICTION, _ALL, _EVENT, _SIGN
+from .moment import _GROUP_ID, _LABEL, _PREDICTION, _ALL, _EVENT, _SIGN, _POSITIVE_UTILITY, \
+    _NEGATIVE_UTILITY
+from fairlearn._input_validation import _MESSAGE_RATIO_NOT_IN_RANGE, _UTILITY_NOT_SPECIFIED
```

Comment:
Let's rename `_POSITIVE_UTILITY` -> `_UTILITY_FOR_1` and `_NEGATIVE_UTILITY` -> `_UTILITY_FOR_0` or something like that. Using positive and negative in this context is a bit confusing.